lucet-runtime/src/c_api.rs

```diff
@@ -420,9 +420,7 @@ lucet_hostcalls! {
         vmctx.instance()
             .module()
             .get_func_from_idx(table_idx, func_idx)
-            // the Rust API actually returns a pointer to a function pointer, so we want to dereference
```

Comment:
I didn't see any uses of this function in `lucet` but I think this would have resulted in the same incorrect double-deref of `fptr` for users of the C api?

Reply:
a good reminder that we need a better test suite for the C API